pkg/virt-controller/watch/vm.go

```diff
@@ -2908,7 +2972,7 @@ func (c *VMController) addRestartRequiredIfNeeded(lastSeenVMSpec *virtv1.Virtual
 	// Note: this list needs to stay up-to-date with everything that can be live-updated
 	// Note2: destroying lastSeenVMSpec here is fine, we don't need it later
 	if c.clusterConfig.IsVMRolloutStrategyLiveUpdate() {
-		lastSeenVMSpec.Template.Spec.Volumes = vm.Spec.Template.Spec.Volumes
+		liveUpdateVolumes(lastSeenVMSpec, vm)
```

Comment:
Can you rename this to something like `filterLiveUpdatableVolumes`? Otherwise, I read it as if the function actually handles the live update.